src/data_hub/api.py

```diff
@@ -861,14 +861,22 @@ def get_dataframe(
 
         # this import is here because we don't want to
         # import pandas unless we actually use this function
-        import pandas as pd
+        from deeporigin.data_hub.dataframe import DataFrame
 
-        df = pd.DataFrame(data)
+        df = DataFrame(data)
         df.attrs["file_ids"] = list(set(file_ids))
         df.attrs["reference_ids"] = list(set(reference_ids))
         df.attrs["id"] = database_id
```

Comment:
Is this redundant with the information in `db_row` and therefore df.attrs["metadata"]?

Reply:
there's a lot of redundancy here. i'll clean up in a future pr